site/static/compare.html

```diff
@@ -118,6 +118,7 @@
             margin: 0;
             padding: 0;
         }
+
```

Comment:
Sorry for the formatting changes being included in here. Let me know if this is too much a bother, and I can try to remove them.